src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -3703,13 +3703,146 @@ _test_sample_versioned_api_example_4 (void)
    mongoc_server_api_destroy (server_api);
 }
 
+static int64_t iso_to_unix (const char* iso_str) {
+   /* TODO (CDRIVER-2945) there is no convenient helper for converting ISO8601
+    * strings to Unix timestamps. This is not shown in the example. */
+   return 1628330345;
+}
+
+static void _test_sample_versioned_api_example_5_6_7_8 (void) {
+#define N_DOCS 8
```

Comment:
It looks like this is only used once; do we need this macro?

Reply:
I think so. `N_DOCS` is also used on the `mongoc_collection_insert_many` call on L3786, the assertions on L3818 and L3824, and the cleanup on L3829.